### PR TITLE
improvement: Update node version engine in package settings #155

### PR DESCRIPTION
**Pull request by jpcmf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "prettier": "^2.5.0"
   },
   "engines": {
-    "node": "16",
+    "node": ">=16.19.0 <=18.x.x",
```
Comment (Lith) [inline package.json:58]:
```suggestion
    "node": ">=16.x.x <=18.x.x",
```

**Review by Lith — CHANGES_REQUESTED** [review]
I suggest to allow all node version between 16 and 18

**Comment by jpcmf** [comment]
> I suggest to allow all node version between 16 and 18

Sounds good @Lith!

**Review by Lith — APPROVED** [review]

**Comment by Lith** [comment]
Thanks, sorry about that, but we've missed to Signing your commit, and I can rebase & merge without :( 
You will find more information here : https://docs.github.com/articles/about-gpg/

**Comment by jpcmf** [comment]
> Thanks, sorry about that, but we've missed to Signing your commit, and I can rebase & merge without :( You will find more information here : https://docs.github.com/articles/about-gpg/

Please @Lith, check if everything is ok now.

File: package.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
     "prettier": "^2.5.0"
   },
   "engines": {
-    "node": "16",
+    "node": ">=16.x.x <=18.x.x",
     "npm": "8"
```
Comment (JohnGale87) [inline package.json:59]: Per my comment in the issue:

```suggestion
    "npm": ">=8"
```

Comment (JohnGale87) [reply package.json:59]: https://github.com/strapi-community/strapi-provider-upload-google-cloud-storage/issues/155#issuecomment-1412832543

**Comment by Lith** [comment]
I will merge manually this PR because of GPG restriction, this merging is blocked from GitHub.

**Comment by nielsbox** [comment]
> I will merge manually this PR because of GPG restriction, this merging is blocked from GitHub.
Hi @Lith when will you merge this? Really blocking...

**Pull request closed without merging**